gradle/documentation/documentation.gradle

```diff
@@ -34,4 +36,11 @@ configure(subprojects.findAll { it.path == ':lucene' || it.path == ':solr' }) {
   ext {
     docroot = "${project.buildDir}/documentation"
   }
+  
+  task copyDocumentationAssets(type: Copy) {
+    includeEmptyDirs = false
+    from('site/html')      // lucene
```

Comment:
I'd rather have it as separate configure(":lucene"), configure(":solr") blocks... would be easier to separate the logic between the two?

Reply:
I was thinking about that too, but this was easier to begin with. I did not want to clone the whole "Copy" task several times. If you tell me how to do this easier, give me a hint!

Should I remove the `from` here and then add 2 separate configures for lucene and solr just adding the `from`?

Reply:
I did this for the main task. At this place we can also do it, but that's less to maintain.